extensions/smallrye-reactive-messaging/deployment/src/main/java/io/quarkus/smallrye/reactivemessaging/deployment/devconsole/ReactiveMessagingDevUiProcessor.java

```diff
@@ -0,0 +1,28 @@
+package io.quarkus.smallrye.reactivemessaging.deployment.devconsole;
+
+import io.quarkus.deployment.IsDevelopment;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.devui.spi.JsonRPCProvidersBuildItem;
+import io.quarkus.devui.spi.page.CardPageBuildItem;
+import io.quarkus.devui.spi.page.Page;
+import io.quarkus.smallrye.reactivemessaging.runtime.devconsole.ReactiveMessagingJsonRpcService;
+
+public class ReactiveMessagingDevUiProcessor {
+
+    @BuildStep(onlyIf = IsDevelopment.class)
+    CardPageBuildItem create() {
+        CardPageBuildItem card = new CardPageBuildItem("SmallRye Reactive Messaging");
+        card.addPage(Page.webComponentPageBuilder()
+                .title("Channels")
+                .componentLink("qwc-smallrye-reactive-messaging-channels.js")
+                .icon("font-awesome-solid:diagram-project"));
+
+        return card;
+    }
+
+    @BuildStep(onlyIf = IsDevelopment.class)
+    JsonRPCProvidersBuildItem createJsonRPCServiceForCache() {
+        return new JsonRPCProvidersBuildItem("SmallRyeReactiveMessaging",
```

Comment:
Yea, here again I would suggest `-` in the place of space, or just the space.

Reply:
I'm still puzzled by this convention. Why "snake" ? and as I mentioned, I would avoid spaces in identifiers, really error prone (don't you know yaml? )

Reply:
So I am already converting that to lower case and dash for the url, so that would be the best. Maybe for both the card and this. I would like to keep them the same.

We can discuss that separately and do a PR to conform later.